db/osqlcomm.c

```diff
@@ -7297,14 +7271,17 @@ int osql_process_packet(struct ireq *iq, unsigned long long rqid, uuid_t uuid,
 static int sorese_rcvreq(char *fromhost, void *dtap, int dtalen, int type,
                          int nettype)
 {
-    osql_sess_t *sess = NULL;
-    uint8_t *malcd = malloc(OSQL_BP_MAXLEN);
+    int rc = 0;
     struct ireq *iq = NULL;
+    uint8_t *malcd = malloc(OSQL_BP_MAXLEN);
```

Comment:
32k - we can probably just allocate on stack `uint8_t malc[OSQL_BP_MAXLEN]` instead of malloc, free.